docker/docs/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:latest
+RUN apt-get update
+RUN apt-get install apt-utils
+RUN apt-get install -y tzdata && dpkg-reconfigure --frontend noninteractive tzdata
+RUN apt-get install -y build-essential cmake git libjson-c-dev uuid-dev git python3-dev python3-wheel python3-pip doxygen
+WORKDIR /root
+ENTRYPOINT [ "/bin/bash", "-c" ]
```

Comment:
Doesn't -c require a command to execute, or is it different in bash.  I know that sh -c requires a command.

Reply:
It requires a command, the point of this container's entry point is basically to run a bash command in the context of the container.